src/app/auth/register/gravatar-info/gravatar-info.component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, OnInit, Input } from '@angular/core';
 import * as md5 from 'md5';
 import { FormGroup } from '@angular/forms';
-import { User } from '@cyber4all/clark-entity';
+import { User } from '../../../../entity/index';
```

Comment:
This should be able to be 
```suggestion
import { User } from '../../../../entity';
```
because of the index file being there.

Reply:
If we actually go with the type alias, this won't matter.

Reply:
Great idea on adding the path so we can use the alias ```@entity```. With that addition this shouldn't be necessary then. Would it be cleaner to reference entity as such ```"@entity": ["entity/index"]``` or still remove index